app/code/local/Itransition/ShippingInsurance/Block/System/Config/Form/Field/Custom.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class Itransition_ShippingInsurance_Block_System_Config_Form_Field_Custom
+    extends Mage_Adminhtml_Block_System_Config_Form_Field
+    implements Varien_Data_Form_Element_Renderer_Interface
+{
+    public function render(Varien_Data_Form_Element_Abstract $element)
+    {
+        $prototype = $element->getData('field_config');
+
+        if ($prototype && $prototype instanceof Mage_Core_Model_Config_Element && isset($prototype->label)) {
+            $element->setLabel($prototype->label);
```

Comment:
Magic explanation: $prototype is an XML node which declared in system.xml. Label in prototype changed due clone strategy (get prefixes) in `Itransition_ShippingInsurance_Model_Setting_Cloned`. This condition revert label back from original XML. 
Sorry, may be exists better solution, but it works :)